packages/runtime/container-runtime/src/batchManager.ts

```diff
@@ -21,16 +23,19 @@ export type BatchMessage = IBatchMessage & {
 export class BatchManager {
     private pendingBatch: BatchMessage[] = [];
     private batchContentSize = 0;
+    private outOfOrderMessagesToReport = 30;
```

Comment:
I think 30 is too much :) I think 5 is more than enough - most sessions have less than 50 events (I think), so adding 30 is a lot! :)